### ISSUE TITLE
CA1727 is not reported for LoggerMessageAttribute

### ISSUE THREAD

**Issue by DeKon0650** [dotnet/roslyn-analyzers#6051]
### Analyzer

**Diagnostic ID**: [CA1727 ](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/CA1727): `Use PascalCase for named placeholders`

### Analyzer source

**SDK**: [Built-in CA analyzers in .NET 5 SDK or later](https://docs.microsoft.com/dotnet/fundamentals/productivity/code-analysis)

**Version**: [SDK 6.0.301](https://dotnet.microsoft.com/download/dotnet/6.0)

### Describe the bug

CA1727 is not reported for messages in LoggerMessageAttribute

### Steps To Reproduce

1. Make sure that CA1727 is enabled, e.g. by using `<AnalysisMode>AllEnabledByDefault</AnalysisMode>`
2. Write a line like `logger.LogError("Unsuccessful status code {statusCode}");`
3. Notice that CA1727 is shown.
4. Remove the line and write a partial method with LoggerMessageAttribute like
```
[LoggerMessage(-1, LogLevel.Error, "Unsuccessful status code {statusCode}")]
private partial void LogUnsuccessfulStatusCode(HttpStatusCode statusCode);
```
5. Notice that the warning is not shown any more.

### Expected behavior

CA1727 should be shown.

### Actual behavior

CA1727 is not shown.
